app/src/main/java/com/depromeet/team6/presentation/ui/course/component/TransportCourseInfo.kt

```diff
@@ -0,0 +1,131 @@
+package com.depromeet.team6.presentation.ui.course.component
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.depromeet.team6.R
+import com.depromeet.team6.presentation.model.course.TransportCourseInfo
+import com.depromeet.team6.presentation.model.course.TransportType
+import com.depromeet.team6.ui.theme.defaultTeam6Colors
+
+@Composable
+fun TransportCourseInfoExpandable(
+    transportCourseInfo: List<TransportCourseInfo>,
+    modifier: Modifier = Modifier
+) {
+    val context = LocalContext.current
+    var expanded by remember { mutableStateOf(false) }
```

Comment:
```suggestion
    var expanded by rememberSaveable { mutableStateOf(false) }
```

rememberSaveable 을 사용하여 관리하면 Activity 재생성시에도 상태가 유지되기 때문에 안정성 측면에서 좋을 것 같아요!

Reply:
remember에서는 상태가 RAM 메모리에 저장되는 반면 rememberSaveable은 Bundle에 저장하는거로 알고있습니다!
애니메이션같은 경우는 데이터 유지보단 가볍고 빠른 속도가 더 중요하다 판단해서 remember를 사용했어요

반면에 텍스트 필드나 체크박스 같은 뷰의 경우는 데이터 유지가 더 중요하기 때문에 rememberaSaveable를 쓰면 좋을것 같습니다 :)